### PR TITLE
[PAY-808] [PAY-810] Float new playlists to top of library

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

- New playlists start on top of playlist library in navbar
- Changed default sort order to recency in profile page
- Fixed issues with playlist creation:
Prior to this PR, creating a playlist would write out a playlist to your account's collections, which would get flushed to local storage. The playlist library wouldn't get updated until *next session* as part of the sign on sagas, which check for playlist library state differing from account collection state. This PR writes out the playlist library as part of the confirmation flow.

### Dragons

Playlist creation flow is a bit scary. But this seems to work

### How Has This Been Tested?

Web. Testing mobile

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/piazz-fix-playlist-ordering

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/piazz-fix-playlist-ordering

File: packages/web/src/common/store/cache/collections/sagas.js [hunk 1]
```diff
@@ -234,6 +235,9 @@ function* confirmCreatePlaylist(uid, userId, formFields, source) {
           })
         )
 
+        // Write out the new playlist to the playlist library
+        yield call(addPlaylistsNotInLibrary)
```
Comment (dylanjeffers) [inline packages/web/src/common/store/cache/collections/sagas.js:239]: just want to understand flow between this call, and the upload sagas one. will they not get called back-to-back?

**Review by dylanjeffers — APPROVED** [review]
sweet!

File: packages/web/src/common/store/cache/collections/sagas.js (hunk 1 above)
Comment (piazzatron) [reply packages/web/src/common/store/cache/collections/sagas.js:239]: you would think so, and I did as well - however, it looks like the upload sagas does the same work create playlist does (it calls createPlaylist, etc), so we need to treat them distinctly

**Pull request merged**